From ecda3b09ab22e8b2bcd14204560f2b5975164e48 Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Fri, 21 Mar 2025 08:33:23 +0100
Subject: [PATCH] fix

---
 src/common/root/.curlrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/root/.curlrc b/src/common/root/.curlrc
index 0726695..4c31e5f 100644
--- a/src/common/root/.curlrc
+++ b/src/common/root/.curlrc
@@ -1,3 +1,3 @@
 # See https://curl.haxx.se/docs/ssl-ciphers.html for the NSS names
-ciphers = ecdhe_rsa_aes_256_sha_384,ecdhe_rsa_aes_256_gcm_sha_384,rsa_aes_256_cbc_sha_256,rsa_aes_128_cbc_sha_256,ECDHE-ECDSA-AES128-GCM-SHA256
+ciphers = ECDHE-RSA-AES256-SHA384,ECDHE-RSA-AES256-GCM-SHA384,AES256-SHA256,AES128-SHA256,ECDHE-RSA-AES128-GCM-SHA256,ECDHE-ECDSA-AES128-GCM-SHA256
 tlsv1.2
-- 
2.47.0

