diff -NaurZ xcp-ng-release-8.2.0-copy/src/common/sbin/update-issue xcp-ng-release-8.2.0/src/common/sbin/update-issue
--- xcp-ng-release-8.2.0-copy/src/common/sbin/update-issue	2021-01-26 09:29:14.011580635 +0100
+++ xcp-ng-release-8.2.0/src/common/sbin/update-issue	2021-01-26 09:31:30.669856728 +0100
@@ -28,7 +28,7 @@


 [ -z "$MANAGEMENT_INTERFACE" ] || {
-   IP=$(ip addr show dev $MANAGEMENT_INTERFACE | awk '/ inet / { print $2 }')
+   IP=$(ip addr show dev $MANAGEMENT_INTERFACE | awk '/ inet6? / { print $2 }')
    IP=${IP%%/*}
 }
 : ${IP:="IP address not configured"}
diff --git a/src/common/etc/sysctl.d/90-net.conf b/src/common/etc/sysctl.d/90-net.conf
index dee5325f..3cd338dc 100644
--- a/src/common/etc/sysctl.d/90-net.conf
+++ b/src/common/etc/sysctl.d/90-net.conf
@@ -22,7 +22,3 @@ net.ipv4.conf.all.arp_announce = 1
 net.bridge.bridge-nf-call-iptables = 0
 net.bridge.bridge-nf-call-ip6tables = 0
 net.bridge.bridge-nf-call-arptables = 0
-
-# Disable IPv6 on interfaces
-net.ipv6.conf.all.disable_ipv6 = 1
-net.ipv6.conf.default.disable_ipv6 = 1
diff --git a/src/common/etc/sysctl.d/91-net-ipv6.conf b/src/common/etc/sysctl.d/91-net-ipv6.conf
new file mode 100644
index 00000000..4f1b3d51
--- /dev/null
+++ b/src/common/etc/sysctl.d/91-net-ipv6.conf
@@ -0,0 +1,3 @@
+# Disable IPv6 on interfaces
+net.ipv6.conf.all.disable_ipv6 = 1
+net.ipv6.conf.default.disable_ipv6 = 1
diff --git i/xcp-ng.repo w/xcp-ng.repo
index 0259a92..7d200a2 100644
--- i/xcp-ng.repo
+++ w/xcp-ng.repo
@@ -22,3 +22,10 @@ gpgcheck=1
 repo_gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-xcpng

+[xcp-ng-ipv6]
+name=XCP-ng IPv6 Repository
+baseurl=http://mirrors.xcp-ng.org/8/8.2/ipv6/x86_64/ http://updates.xcp-ng.org/8/8.2/ipv6/x86_64/
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-xcpng
